src/engine/RuntimeInformation.cpp

```diff
@@ -106,8 +106,9 @@ double RuntimeInformation::getOperationTime() const {
   } else {
     auto result = totalTime_;
     for (const RuntimeInformation& child : children_) {
-      // The `totalTime_` does not account for the time spent during
-      // query planning.
+      // If a child was computed during the query planning, the time spent
+      // computing that child is *not* included in this operation's
+      // `totalTime_`, That's why we skip such children in the following loop.
```

Comment:
Much better, thanks a lot! Minor TODO: the comma before "That's" should be a full stop and the comment should be moved to after line 106 (it refers to "the following loop", so it shouldn't be in the loop).